test/project_types/script/layers/infrastructure/script_service_test.rb

```diff
@@ -55,11 +55,6 @@
     HERE
   end
 
-  before do
-    ::Script::Layers::Infrastructure::ScriptService::MakeRequest.stubs(:bypass_partners_proxy).returns(false)
-    # script_service.stubs(:bypass_partners_proxy).returns(false)
-  end
-
```

Comment:
This was removed after Max and I discovered it didn't actually do anything, it basically is stubbing in what would be returned anyway (you can verify this by commenting it out in the master branch and running the tests).  I removed it here since I would had to stub out some method on `ApiClients` after the move.

The tests were written against `PartnersProxyAPI`, you can see that it's stubbing out things like `load_query` and what not. The tests can simplified now that the only thing we would need to stub out is the response from a duck type for `query()`.

Reply:
Right, OK.